web/index.html

```diff
@@ -1185,7 +1185,7 @@ <h1>Accedian Skylight API Portal</h1>
                             analytics to engineering, operations and planning teams</p>
                         <div style="display: flex;">
                             <a class="btn -gold js-scroll-to" href="https://api.analytics.accedian.io/session.html" target="_blank">Go to Skylight Performance Analytics API</a>
-                            <a class="btn -grey js-scroll-trigger" href="./sensors/v0.5/functions" style="margin-left: 12px;">Go to Skylight PVQL API</a>
+                            <a class="btn -grey js-scroll-trigger" href="./sensors/v0.7/functions" style="margin-left: 12px;">Go to Skylight PVQL API</a>
```

Comment:
Also, if we don't provide links to older versions, does it make sense to include them?

Reply:
I don't think we provide earlier documentation, however it seems that we deliberately set the version of the API  as we store the documentation in a `v0.5` prefix. So don't really know if we plan or not to have a documentation per version,